src/htmlEntities.ts

```diff
@@ -0,0 +1,6 @@
+const entities: { [key: string]: string } = {
+  '<': '&lt;',
+  '>': '&gt;',
+};
+
+export const encodeHtml = (str: string) => str.replace(/[&<>'"]/g, tag => entities[tag] || tag);
```

Comment:
Why do we have to use `&,',"` if we return them?

Reply:
I modified another script to make it simpler since we don't care about those other characters, but forgot to modify the regex, will fix

Reply:
I modified another script to make it simpler since we don't care about those other characters, but forgot to modify the regex, will fix

Reply:
```suggestion
export const encodeHtml = (str: string) => str.replace(/[<>]/g, tag => entities[tag] || tag);
```

Reply:
```suggestion
export const encodeHtml = (str: string) => str.replace(/[<>]/g, tag => entities[tag] || tag);
```